docs/deploy.sh

```diff
@@ -11,14 +11,42 @@ if [ ! -d "$target" ]; then
   exit 1
 fi
 
-aws s3 sync "$target" "s3://${bucket}" --delete
+# To keep the previous website up and running, we deploy it using these steps.
+#   1. Upload JS files in \`_next/**\` folder
+#      > Uploads the new generated JS files, containing hashes to not-collide with previous deployment
+#   2. Upload new asset files in \`static/**\` folder
+#      > Contains large files and might slow down overwrite of HTML files
+#   3. Sync assets and clean up outdated files from previous deployments
+#   4. Add custom redirects
 
+# Due to a bug with `aws s3 sync` we need to copy everything first instead of syncing
+# see: https://github.com/aws/aws-cli/issues/3273#issuecomment-643436849
+
+echo "::group::[1/4] Upload JS files in \`_next/**\` folder"
 aws s3 cp \
+  --no-progress \
   --recursive \
-  --metadata-directive REPLACE \
-  --cache-control "public,max-age=31536000,immutable" \
-  "s3://${bucket}/_next/static/" \
-  "s3://${bucket}/_next/static/"
```

Comment:
This static folder contains the JS files of the pages. Looking at your comments James, I'm not sure if we should keep this. It doesn't contain assets like images or videos.